source/extensions/omni.isaac.lab/test/sensors/test_camera.py

```diff
@@ -231,6 +231,57 @@ def test_multi_camera_init(self):
                 for im_data in cam.data.output.to_dict().values():
                     self.assertEqual(im_data.shape, (1, self.camera_cfg.height, self.camera_cfg.width))
 
+    def test_camera_init_intrinsic_matrix(self):
+        # get the first camera
+        camera_1 = Camera(cfg=self.camera_cfg)
+        # get intrinsic matrix
+        self.sim.reset()
+        intrinsic_matrix = camera_1.data.intrinsic_matrices[0].cpu().flatten().tolist()
+        self.tearDown()
+        # reinit the first camera
+        self.setUp()
+        camera_1 = Camera(cfg=self.camera_cfg)
+        # initialize from intrinsic matrix
+        intrinsic_camera_cfg = CameraCfg(
+            height=128,
+            width=128,
+            prim_path="/World/Camera_2",
+            update_period=0,
+            data_types=["distance_to_image_plane"],
+            spawn=sim_utils.PinholeCameraCfg().from_intrinsic_matrix(
+                intrinsic_matrix=intrinsic_matrix,
+                width=128,
+                height=128,
+                focal_length=24.0,
+                focus_distance=400.0,
+                clipping_range=(0.1, 1.0e5),
+            ),
+        )
+        camera_2 = Camera(cfg=intrinsic_camera_cfg)
+
+        # play sim
+        self.sim.reset()
+        self.sim.play()
```

Comment:
Why do you need to do `sim.play()` again? `sim.reset()` does it for you from what I know.